binsync/decompilers/angr/controller.py

```diff
@@ -187,6 +190,10 @@ def fill_function_header(self, func_addr, user=None, artifact=None, decompilatio
                         changes = True
         return changes
 
+    def remove_unused_structs(self):
```

Comment:
Same here. Since this is a Public API function, it needs to be more explicit on what it does. What is this removing structs from? Alternatively, you can make this a private function so it can be less descriptive.